backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java

```diff
@@ -718,10 +716,10 @@ public void requestRendering () {
 
 	@Override
 	public boolean isFullscreen () {
-		return true;
+		Activity activity = app.getActivity();
+		return activity == null || Build.VERSION.SDK_INT < 24 || !activity.isInMultiWindowMode();
```

Comment:
Not very readable in my opinion but I accept it's a matter of personal taste

Reply:
I would probably put brackets ( ... ) around the sub-conditions, but maybe I am little bit strict there too :)